be/src/Project/Services/Identity/Identity.Api/Areas/Home/ModelBuilders/PrivacyPolicyPageModelBuilder.cs

```diff
@@ -14,32 +16,33 @@ public class PrivacyPolicyPageModelBuilder : IAsyncComponentModelBuilder<Compone
         private readonly IAsyncComponentModelBuilder<ComponentModelBase, MetadataViewModel> _seoModelBuilder;
         private readonly IModelBuilder<HeaderViewModel> _headerModelBuilder;
         private readonly IModelBuilder<FooterViewModel> _footerModelBuilder;
-        private readonly IContentRepository _contentRepository;
+        private readonly IPolicyRepository _policyRepository;
 
         public PrivacyPolicyPageModelBuilder(
             IAsyncComponentModelBuilder<ComponentModelBase, MetadataViewModel> seoModelBuilder,
             IModelBuilder<HeaderViewModel> headerModelBuilder,
             IModelBuilder<FooterViewModel> footerModelBuilder,
-            IContentRepository contentRepository)
+            IPolicyRepository policyRepository)
         {
             _seoModelBuilder = seoModelBuilder;
             _headerModelBuilder = headerModelBuilder;
             _footerModelBuilder = footerModelBuilder;
-            _contentRepository = contentRepository;
+            _policyRepository = policyRepository;
         }
 
         public async Task<PrivacyPolicyPageViewModel> BuildModelAsync(ComponentModelBase componentModel)
         {
-            var content = await _contentRepository.GetContentAsync(componentModel.ContentPageKey, componentModel.Language);
+            var policy = await _policyRepository.GetPolicyAsync(componentModel.Language);
 
             var viewModel = new PrivacyPolicyPageViewModel
             {
                 Metadata = await _seoModelBuilder.BuildModelAsync(componentModel),
                 Header = _headerModelBuilder.BuildModel(),
-                Content = new ContentPageViewModel
+                Policy = new PolicyDetailViewModel
```

Comment:
**suggestion:** let's use PrivacyPolicy instead of Policy alone everywhere for consistency